src/libs/actions/IOU.ts

```diff
@@ -969,6 +969,33 @@ function createDistanceRequest(
     Report.notifyNewAction(chatReport.reportID, userAccountID);
 }
 
+/**
+ * Compute the diff amount when we update the transaction
+ */
+function computeDiffAmount(iouReport: OnyxEntry<OnyxTypes.Report>, updatedTransaction: OnyxEntry<OnyxTypes.Transaction>, transaction: OnyxEntry<OnyxTypes.Transaction>): number {
+    if (!iouReport) {
+        return 0;
+    }
+    const isExpenseReport = ReportUtils.isExpenseReport(iouReport);
+    const updatedCurrency = TransactionUtils.getCurrency(updatedTransaction);
+    const currentCurrency = TransactionUtils.getCurrency(transaction);
+
+    if (updatedCurrency === iouReport?.currency && currentCurrency !== iouReport?.currency) {
+        // add the diff with the total if we change the currency from different currency to the currency of the iou report
```

Comment:
If English is not your native language, it can be very hard to use "a" and "the" correctly. In general, we use "a"/"an" if we talk about an entity/person/object from a group/set/category, without having a specific instance in mind (jn the given context). We use "the" when we talk about a specific instance (in the given context).